kv/txn_correctness_test.go

```diff
@@ -310,15 +310,17 @@ func TestEnumerateIsolations(t *testing.T) {
 		{SSI, SI, SI},
 		{SI, SI, SI},
 	}
-	if !reflect.DeepEqual(enumerateIsolations(3, bothIsolations), expIsolations) {
-		t.Errorf("expected enumeration to match %s; got %s", expIsolations, enumerateIsolations(3, bothIsolations))
+	enum := enumerateIsolations(3, bothIsolations)
+	if !reflect.DeepEqual(enum, expIsolations) {
```

Comment:
can you make this `if enum := ...; !reflect.DeepEqual(...) {` and the same below? sorry, I missed it below.